modules/geo-layers/src/terrain-layer/terrain-layer.js

```diff
@@ -203,7 +203,14 @@ export default class TerrainLayer extends CompositeLayer {
       maxZoom,
       minZoom,
       extent,
-      maxRequests
+      maxRequests,
```

Comment:
Should these be declared in prop types table? Should they be documented?

Reply:
Documentation:

> When in Tiled Mode, inherits from all TileLayer properties

Reply:
> Should these be declared in prop types table

https://github.com/visgl/deck.gl/blob/3a00024c4ee558f66a684710c34c98309e5917f8/modules/geo-layers/src/terrain-layer/terrain-layer.js#L31-L32

Is this what you mean?

> Should they be documented?

https://deck.gl/docs/api-reference/geo-layers/terrain-layer#properties

> When in Tiled Mode, inherits from all TileLayer properties

It already is documented, but the documentation is currently wrong because the props aren't forwarded on.